README.md

```diff
@@ -31,6 +31,8 @@ Monorepo for Streamr Network packages.
 | NodeJS version `16.13.x` and NPM version `8.x` is required |
 | --- |
 
+Installation on m1 Macs requires additional steps, see [install-on-m1.md](/install-on-m1)
```

Comment:
```suggestion
Installation on an M1 Mac requires additional steps, see [install-on-m1.md](/install-on-m1)
```

M1 vs m1 for consistency.

Reply:
Note also that this link will *probably* break when viewed via https://www.npmjs.com/package/streamr-client

If the link is an absolute url it'd work in both places, but you'd want to prematurely link it to the `main` tree.  

e.g. https://github.com/streamr-dev/network-monorepo/tree/main/network-monorepo/install-on-m1.md

Can't be the only people with this problem though perhaps npm has some automatic magic specifically for transforming github links to work on npm.com, or perhaps there's some other tool/service for doing a buildtime transform or live redirect.